git-hub

```diff
@@ -761,6 +761,7 @@ class CloneCmd (object):
 		if upstream.endswith('.git'): upstream = upstream[:-4]
 		if '/' in upstream:
 			(owner, proj) = upstream.split('/')[-2:]
+			upstream = '{}/{}'.format(owner, proj)
```

Comment:
This eventually should get a comment, it is already confusing enough this split and rejoin without explaining much about what is it supposed to do. It took me a few minutes to figure out why we were splitting and then re-joining the `upstream` string.